### PR TITLE
Add submit and schedule kebechet methods

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies
https://github.com/thoth-station/mi-scheduler/pull/58

## This introduces a breaking change

- [ ] Yes
- [X] No

## This Pull Request implements
New submit and schedule methods for MI inspection of Kebechet 

## Description
the `submit_mi` and `schedule_mi_workflow` methods regarding general mi analysis remained untouched

### CODE REVIEW

**Review by fridex — APPROVED** [review]
:ship: :+1:

File: thoth/common/openshift.py [hunk 1]
```diff
@@ -1643,6 +1661,10 @@ def get_mi_repositories_and_organizations(self,) -> Tuple[List[str], List[str]]:
         )
         return (repos, orgs)
 
+    def get_mi_kebechet_repositories(self):
```
Comment (pacospace) [inline thoth/common/openshift.py:1664]: This method is not taken from common, it is a query from the database that needs to be created :)

Comment (xtuchyna) [reply thoth/common/openshift.py:1664]: yes, but cannot the query be done inside common using e.g. `psycopg2` module? (I am still trying to recreate the postgresql database locally...)

Comment (pacospace) [reply thoth/common/openshift.py:1664]: common and storages are two different libraries and their behavior is separated, moreover, storages already depend on thoth-common, you would create a cycle in dependencies that cannot be solved. We should use libraries for their purposes. common to handle kubernetes, openshift and storages to handle database. 

The component that wants to schedule something uses common, the component the wants to query something uses storages. So in the component both libraries need to be direct dependencies :)

Comment (pacospace) [reply thoth/common/openshift.py:1664]: > yes, but cannot the query be done inside common using e.g. `psycopg2` module? (I am still trying to recreate the postgresql database locally...)

What issues are you experiencing?

Comment (xtuchyna) [reply thoth/common/openshift.py:1664]: > The component that wants to schedule something uses common, the component the wants to query something uses storages. So in the component both libraries need to be direct dependencies :)

yes alright, will move the method to storages then :+1:

File: thoth/common/openshift.py [hunk 2]
```diff
@@ -1427,6 +1427,24 @@ def schedule_mi_workflow(
             },
         )
 
+    def schedule_mi_kebechet_workflow(
```
Comment (pacospace) [inline thoth/common/openshift.py:1430]: can't we extend existing method, to pass the parameter we want? I feel like mi_workflow is one and we can provide specific parameter to do something specific, WDYT?

Comment (xtuchyna) [reply thoth/common/openshift.py:1430]: yes, that's correct. Will do that

Comment (xtuchyna) [reply thoth/common/openshift.py]: Aight, pls have a look at PR and discussion here: https://github.com/thoth-station/thoth-application/pull/692#discussion_r536031962

Kebechet workflows will be then submitted just from `mi-scheduler`

File: thoth/common/openshift.py [hunk 3]
```diff
@@ -1410,14 +1410,22 @@ def schedule_qebhwt_workflow(
         )
 
     def schedule_mi_workflow(
-        self, repository: str, *, job_id: Optional[str] = None,
+        self,
+        repository: str,
+        entities: Optional[str] = None,
```
Comment (pacospace) [inline thoth/common/openshift.py:1415]: please adjust docstring with inputs accordingly :)

**Review by pacospace — COMMENTED** [review]
One comment , otherwise LGTM!

File: thoth/common/openshift.py (hunk 3 above)
Comment (xtuchyna) [reply thoth/common/openshift.py:1415]: done!

**Review by pacospace — APPROVED** [review]
Thanks!!

**Pull request merged**